### ISSUE TITLE
Do I need arduino to control the dc motor and direction?

### ISSUE THREAD

**Issue by tham1996** [hamuchiwa/AutoRCCar#72]
I'm a newbie at E&E and try to learn something. However I'm looking at the coding for a long time and I'm wondering do I need the arduino uno to control the rc car. Thanks in advance.

**Comment by lakshand8** [comment]
No, i used the adafruit motor HAT with the raspberry pi to run the motors using just the keyboard. Theres a bit of code change but its easy to understand once you try a few tutorials from adafruit.

**Comment by datadrian** [comment]
any chance you would share your code?
